sources/europe/de/Brandenburg-DOP20c.geojson

```diff
@@ -0,0 +1,1564 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "Brandenburg-DOP20c",
+    "name": "Brandenburg GeoBasis-DE/LGB (2020-2022) / DOP20c",
```

Comment:
```suggestion
    "name": "Brandenburg GeoBasis-DE/LGB (latest) / DOP20c",
```

See below. Having a year in the name will be out of date every year.

FYI, JOSM has an outdated year for that reason, ATM (https://josm.openstreetmap.de/wiki/Maps/Germany).

Reply:
Change applied in https://github.com/osmlab/editor-layer-index/pull/2109